### ISSUE TITLE
Some (new) questions about features

### ISSUE THREAD

**Issue by GrininViktor** [KaiyangZhou/deep-person-reid#343]
Thank you for your library and OSNet paper!

I have three questions about features. First, can I sum up the feature vectors from previous person appearances (or from different views) to use the result as the most general description of a person? Second, does it make sense to extract features from boxes with only a human mask(without a background)? Third, if i have two similar persons should I include in train only images where I can discern them with my eyes?

**Comment by KaiyangZhou** [comment]
> First, can I sum up the feature vectors from previous person appearances (or from different views) to use the result as the most general description of a person?

Yes you can do this at least you're using the same network so the feature dimensions match. Maybe using some advanced feature encoding method could further improve the performance, e.g. fisher vector encoding.

> Second, does it make sense to extract features from boxes with only a human mask(without a background)?

It makes sense. You might wanna compare the performance between images with and without background, just to confirm it helps.

> Third, if i have two similar persons should I include in train only images where I can discern them with my eyes?

It'd be better to have multiple images for each person, from different views/cameras. You want to let the network learn some useful features by itself.

**Comment by gceboh** [comment]
> Second, does it make sense to extract features from boxes with only a human mask(without a background)?

[Person Search via A Mask-Guided Two-Stream CNN Model](https://arxiv.org/pdf/1807.08107.pdf)

[Mask-guided Contrastive Attention Model for Person Re-Identification](http://openaccess.thecvf.com/content_cvpr_2018/papers/Song_Mask-Guided_Contrastive_Attention_CVPR_2018_paper.pdf)
